angular/projects/admin/src/app/admin/document/document-list/document-list.component.html

```diff
@@ -97,7 +97,7 @@
   <mat-paginator #paginator
                  [length]="total_count"
                  [pageIndex]="0"
-                 [pageSize]="100"
-                 [pageSizeOptions]="[100]">
+                 [pageSize]="5"
```

Comment:
Default page size could perhaps be 20 for a bit bigger default view.

Reply:
Resolves on this commit https://github.com/oddbit/tanam/pull/220/commits/a1ceb003af2a1345729dd671627c520dd4d55a4b